data/standards/manage_OpenStudio_Standards.rb

```diff
@@ -143,7 +143,7 @@ def unique_properties(sheet_name)
          when 'climate_zones'
            ['name', 'standard']
          when 'energy_recovery'
-           ['template', 'climate_zone', 'under_8000_hours']
+           ['template', 'climate_zone', 'under_8000_hours', 'nontransient_dwelling']
```

Comment:
OpenStudio_Standards-ashrae_90_1 table was changed. The new column, 'Nontransient Dwelling', in 'Energy Recovery' tab becomes one of search criteria.